app/views/users/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if @user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+    <ul>
+    <% @user.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
+<%= form_for @user do |f| %>
+  <%= label_tag(:username) %>
+  <%= f.text_field :username %>
+  <%= label_tag(:password) %>
+  <%= f.password_field :password %>
+  <%= label_tag(:email) %>
```

Comment:
Instead of using `label_tag` in a `form_for` you could use `f.label` making use of the form's object.